pkgs/stdenv/generic/setup.sh

```diff
@@ -1558,6 +1549,7 @@ genericBuild() {
             configurePhase ${preBuildPhases[*]:-} buildPhase checkPhase \
             ${preInstallPhases[*]:-} installPhase ${preFixupPhases[*]:-} fixupPhase installCheckPhase \
             ${preDistPhases[*]:-} distPhase ${postPhases[*]:-}";
+        echo "phases: $(echo $phases | tr -s ' ')"
```

Comment:
To avoid a subshell, use:
```suggestion
        echo -n "phases: "
        echo $phases | tr -s ' '
```

Reply:
Actually, following works as well and looks better, but thanks for thinking about performance as well

```
echo "phases: $phases" | tr -s ' '
```